modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2RenewMultiplesKeys.testcase

```diff
@@ -0,0 +1,123 @@
+@component-name = "portal-solutions"
+definition {
+
+	property osgi.modules.includes = "osb-site-initializer-customer-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Customer Portal";
+
+	var testSiteName = "Customer Portal Site";
+	var testSiteURL = "customer-portal-site";
+	var testAccount = "TesteremoveAccount";
+	var testProduct = "DXP";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Login on OKTA") {
+			CustomerPortal.loginOktaToUpdateCP();
+		}
+
+		task ("Create a new Customer Portal Site") {
+			CustomerPortal.addCustomerPortalSite(siteName = ${testSiteName});
+		}
+
+		task ("Add test user access to all Accounts") {
+			CustomerPortal.assignUserToOrganization();
+
+			CustomerPortal.addUserToOneOrMoreAccounts(accountsList = ${testAccount});
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		CustomerPortal.customerPortalApisTearDown();
+
+		if (${testLiferayVirtualInstance} == "false") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = ${testSiteName});
+		}
+	}
+
+	@description = "Automation Ticket: LPS-157452 | Test Summary: Assert if the user can renew keys."
+	@ignore = "true"
+	@priority = 4
+	test AssertThatTheUserCanRenewTheKeys {
+		task ("Given the user Role is Changed to Administrator") {
+			CP2Utils.assigneToRoleViaUsersTab(
+				accountRoleTitle = "Administrator",
+				accountUserName = "Test Test",
+				searchItem = "test",
+				testAccount = ${testAccount});
+		}
+
+		task ("And the user go to Test Account") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+
+			CustomerPortal.search(searchItem = ${testAccount});
+
+			CustomerPortal.enterOnProject(title = ${testAccount});
+
+			CP2Utils.onboardingFlow();
+		}
+
+		task ("And go to DXP for generate 2 new keys") {
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = ${testProduct});
+
+			CP2ActivationKeys.fullFlowGenerateKey(
+				fieldText = "RenewKeys",
+				fieldType = "Environment Name",
+				hostName = "renewtest",
+				keyType = "Key Type",
+				option = "Production",
+				optionIndex = 2);
+
+			CustomerPortal.clickAtButton(text = "Generate 1 Key");
+
+			Button.click(button = "Cancel");
+
+			CP2ActivationKeys.fullFlowGenerateKey(
+				fieldText = "RenewKeys2",
+				fieldType = "Environment Name",
+				hostName = "renewtest",
+				keyType = "Key Type",
+				option = "Production",
+				optionIndex = 2);
+
+			CustomerPortal.clickAtButton(text = "Generate 1 Key");
+
+			Alert.viewErrorMessage(errorMessage = "Error:An unexpected error occurred.");
```

Comment:
No, this is a workaround add because of a fail in the automation in this test, but this doesn't go impact the Ci routine of something like that, because this test has @ignored